app/src/main/AndroidManifest.xml

```diff
@@ -13,25 +13,24 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".ViewAnotherUserActivity" />
-        <activity android:name=".InvitationConfirmationActivity" />
-        <activity android:name=".SupervisedGroupPageActivity" />
-        <activity android:name=".SupervisedGroupListActivity" />
-        <activity android:name=".MonitoringGroupPageActivity" />
-        <activity android:name=".MonitoringGroupListActivity" />
-        <activity android:name=".SearchContactsToSupervise" />
-        <activity android:name=".SendSmsInvitationActivity" />
+        <activity android:name=".viewModels.ViewAnotherUserActivity" />
+        <activity android:name=".viewModels.InvitationConfirmationActivity" />
+        <activity android:name=".activityPage.SupervisedGroupPageActivity" />
+        <activity android:name=".viewModels.SupervisedGroupListActivity" />
+        <activity android:name=".activityPage.MonitoringGroupPageActivity" />
+        <activity android:name=".viewModels.MonitoringGroupListActivity" />
+        <activity android:name=".viewModels.SearchContactsToSupervise" />
+        <activity android:name=".viewModels.SendSmsInvitationActivity" />
         <activity android:name=".activityPage.HomePageActivity" />
-        <!-- TODO The login page should load up first until we have a welcome page-->
         <activity android:name=".activityPage.LoginPageActivity"/>
-        <activity android:name=".activityPage.MainActivity">
+        <activity android:name=".activityPage.MainPageActivity"/>
+        <!--The login page should load up first-->
```

Comment:
Simple change. 

- [ ] Remove comment, or make it refer to welcome page instead

Reply:
I can remove the comment. I just wanted to show information why I decided to put the intent-filter there.

Reply:
It's ok to keep as well, I was just saying that either remove it or change it to refer to the correct starting activity

Reply:
@darkheros12 Also, maybe your comment meant that once WelcomeActivity loads, the first activity after it should be LoginActivity. if that's the case, you can resolve this.